src/components/App/index.js

```diff
@@ -45,6 +49,9 @@ class App extends Component {
   }
 
   renderBody() {
+    if (!this.props.requests) {
+      this.props.fetchPrs(this.props.oauthToken, this.props.loggedInUser);
```

Comment:
So funny story, but if we want this in a lifecycle method, it has to be componentDidUpdate.
The reason for this is that the fetch function needs the login details.  These aren't populated yet on initial mount of this component.  They aren't even populated yet on componentDidMount.  They get updated some time later, so we need to try this call once they are up to date.  Luckily, the call will only ever go once anyway, due to the check if we already have the requests populated.
I will push up what this looks like momentarily.